tests/specs/lemmas.k

```diff
@@ -170,17 +172,11 @@ module LEMMAS-JAVA [symbolic, kast]
        andBool ( #sizeByteArray(WS) ==Int 32 orBool #sizeByteArray(WS) ==Int 64 )
       [simplification]
 
-    rule N +Int 0 => N  [simplification]
-
-    rule N -Int 0 => N  [simplification]
-
     rule 1 *Int N => N  [simplification]
     rule N *Int 1 => N  [simplification]
     rule 0 *Int _ => 0  [simplification]
     rule _ *Int 0 => 0  [simplification]
 
-    rule N /Int 1 => N  [simplification]
-
     rule 0 |Int N => N  [simplification]
     rule N |Int 0 => N  [simplification]
     rule N |Int N => N  [simplification]
```

Comment:
I'm approving, but why do we not also move the other arith lemmas into the common one? You only need this one right now?